code/__HELPERS/lists.dm

```diff
@@ -637,3 +637,6 @@ proc/dd_sortedObjectList(list/incoming)
 #define LAZYACCESS(L, I) (L ? (isnum(I) ? (I > 0 && I <= L.len ? L[I] : null) : L[I]) : null)
 #define LAZYLEN(L) length(L)
 #define LAZYCLEARLIST(L) if(L) L.Cut()
+
+#define LAZYQDELLIST(L) if(L) { for(var/o in L) qdel(o); L.Cut() }
+#define LAZYDESTROYLIST(L) if(L) { LAZYQDELLIST(L); L = null }

```

Comment:
Is there ever a time you wouldn't want to `null` the list out, as well? Just wondering if this second define is even needed.